http/submitattestations.go

```diff
@@ -95,7 +95,14 @@ func createUnversionedAttestations(attestations []*spec.VersionedAttestation) ([
 		case spec.DataVersionDeneb:
 			unversionedAttestations = append(unversionedAttestations, attestations[i].Deneb)
 		case spec.DataVersionElectra:
-			unversionedAttestations = append(unversionedAttestations, attestations[i].Electra)
+			singleAttestation, err := attestations[i].Electra.ToSingleAttestation(attestations[i].ValidatorIndex)
+			if err != nil {
+				s.log.Warn().Err(errors.Join(errors.New("failed to convert attestation to single attestation"), err)).
```

Comment:
I don't think you need that Join here, as it contains the same message as you are logging.